.circleci/config.yml

```diff
@@ -61,12 +61,6 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
-      - run: sudo npm install -g ganache-cli@6.4.2
```

Comment:
@NicholasDotSol what do we use now?

Reply:
I believe something in the chain (possibly openzeppelin) just runs this in the same process.